### PR TITLE
feat: add full scan of userAccountControl attribute

### PR DESCRIPTION
**Pull request by voikin**
an error occurred that the value of the userAccountControl attribute is not always equal to 514

so I did a full attribute check and I'm looking for an attribute indicating that the account is blocked

### CODE REVIEW

File: object/ldap_util.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	"RESERVED",
+	"RESERVED",
+	"RESERVED",
+	"RESERVED",
+}
+
+func ConvertUserAccountControl(uac int) []string {
+	var flags []string
+
+	for i := 0; i < len(UacPropertyFlags); i++ {
```
Comment (Aleksgav) [inline object/ldap_util.go:45]: range?

Comment (voikin) [reply object/ldap_util.go:45]: you`re right

Comment (voikin) [reply object/ldap_util.go]: done

File: object/ldap_util.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	"RESERVED",
+	"RESERVED",
+	"RESERVED",
+}
+
+func ConvertUserAccountControl(uac int) []string {
+	var flags []string
+
+	for i, uacFlag := range UacPropertyFlags {
+		if uac&int(math.Pow(2, float64(i))) != 0 {
```
Comment (Aleksgav) [inline object/ldap_util.go:46]: mb:
```
int(math.Pow(2, float64(i)))
```
simplify to:
```
1 << i
```

Comment (voikin) [reply object/ldap_util.go]: done

**Pull request merged**